### PR TITLE
Fix for 'database does not exist' error

### PR DESCRIPTION
**Pull request by ghost**
If you are using a MySQL database, your app will throw you a 'database does not exist' error after you leave it idle for a few hours. Apparently, MySQL times-out the database connection if it doesn't receive a command for a while (mine turned out to be 8 hours).

Rails doesn't have this problem because it auto-verifies the database connection before it runs the controllers, and re-connects if it needs to. I'm adding this exact same behaviour to the library.

TLDR: You need to call 'ActiveRecord::Base.verify_active_connections!'

### CODE REVIEW

**Comment by janko** [comment]
You can download the new version of the gem, I've submitted your commit. I'm the new maintainer now, here's my [fork](https://github.com/janko-m/sinatra-activerecord) :)
